YTPlayerView/YTPlayerView.m

```diff
@@ -114,8 +114,8 @@ - (void)stopVideo {
 - (void)seekToSeconds:(float)seekToSeconds allowSeekAhead:(BOOL)allowSeekAhead {
   NSNumber *secondsValue = [NSNumber numberWithFloat:seekToSeconds];
   NSString *allowSeekAheadValue = [self stringForJSBoolean:allowSeekAhead];
-  NSString *command = [NSString stringWithFormat:@"player.seekTo(%@, %@);", secondsValue, allowSeekAheadValue];
-  [self stringFromEvaluatingJavaScript:command];
+  NSString *command = [NSString stringWithFormat:@"player.seekTo(%@, false);", secondsValue];
```

Comment:
I've changed the `allowSeekAheadValue` to be always `false` because this is what I wanted in my case but it might be better if we use the parameter `allowSeekAheadValue` . 

I've noticed that this parameter is set to `true` in `RCTYouTubeManager.m`. it might be nice to let the user set it in `<YouTube />` component. 
So be able to do something like this: `this.refs.player.seekTo(20, false)` :)